docs/index.rst

```diff
@@ -7,11 +7,10 @@ alchemlyb: the simple alchemistry library
 =========================================
 
 **alchemlyb** is a library for doing alchemical free energy calculations more easily.
-It includes functions for parsing data from formats common to existing MD engines, subsampling these data, and fitting these data with an estimator to obtain free energies.
-These functions are simple in usage and pure in scope, and can be chained together to build customized analyses of data.
+It seeks to provide fexlible building blocks which could be chained together for customised analysis and workflows that use these building bocks to allow foolproof automatic analysis.
```

Comment:
flexible

Instead of "foolproof" (because if you build foolproof tools, the universe builds a better fool — and also because it's not nice to refer to users as "fools") use "robust" and "general". These larger building blocks can also be seen as reference implementations/examples. They have to follow best practices. Remove the "automatic" — that just means different things to different people.